package protoFile.pet.ActivationPetSkill2301;
message ActivationSkillPosRequest{
      required int32 id=1;//½ÇÉ«id
      required int32 petId=2;//³èÎïid
}
message ActivationSkillPosResponse{
      optional bool result=1;
      optional string message=2;
}